examples/example_app/example_app/views.py

```diff
@@ -137,6 +140,16 @@ class ExampleModelUIViewSet(views.NautobotUIViewSet):
         ),
     )
 
+    def get_required_permission(self):
```

Comment:
Can we come up with a pattern that doesn't require *both* `custom_action_permission_map` **and** overriding `get_required_permission`, at least in the case where the `model` isn't different?

Looking at `NautobotViewSetMixin`, I'm actually surprised it doesn't already work that way - `get_action()` now gets the correct verb based on `custom_action_permission_map`, and `get_required_permission()` uses `get_action()` as well?